xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -424,15 +381,24 @@ bool CPVRChannelGroup::UpdateClientPriorities()
 }
 
 /********** getters **********/
-const PVRChannelGroupMember& CPVRChannelGroup::GetByUniqueID(const std::pair<int, int>& id) const
+PVRChannelGroupMember& CPVRChannelGroup::GetByUniqueID(const std::pair<int, int>& id)
 {
   CSingleLock lock(m_critSection);
-  PVR_CHANNEL_GROUP_MEMBERS::const_iterator it = m_members.find(id);
+  auto it = m_members.find(id);
   return it != m_members.end() ?
       it->second :
       CPVRChannelGroup::EmptyMember;
 }
 
+const PVRChannelGroupMember& CPVRChannelGroup::GetByUniqueID(const std::pair<int, int>& id) const
+{
+  CSingleLock lock(m_critSection);
+  auto it = m_members.find(id);
+  return it != m_members.end() ?
+  it->second :
+  CPVRChannelGroup::EmptyMember;
```

Comment:
formatting seams wrong to me should be 
```
return it != m_members.end() 
  ? it->second 
  : CPVRChannelGroup::EmptyMember;
```

Reply:
Good catch. Damn auto indentation. Will change this ofc.